.github/workflows/ci.yml

```diff
@@ -16,8 +19,8 @@ jobs:
             package-lock.json
             packages/*/node_modules
           key: ${{ runner.os }}-${{ hashFiles('**/package.json') }}
```

Comment:
Need to change caching too with npm ci - https://github.com/actions/cache/blob/main/examples.md#node---npm

(`npm ci` always nukes node_modules folder and then installs everything from lockfile)